openshift/kustomize/services/filemonitor/base/deploy.yaml

```diff
@@ -0,0 +1,167 @@
+  selector:
+    name: filemonitor-service
+    part-of: tno
+    component: filemonitor-service
+  strategy:
+    rollingParams:
+      intervalSeconds: 1
+      maxSurge: 25%
+      maxUnavailable: 25%
+      timeoutSeconds: 600
+      updatePeriodSeconds: 1
+    type: Rolling
+  template:
+    metadata:
+      name: filemonitor-service
+      labels:
+        name: filemonitor-service
+        part-of: tno
+        component: filemonitor-service
+    spec:
+      volumes:
+        - name: av-storage
+          persistentVolumeClaim:
+            claimName: av-storage
+        - name: ssh-key 
+          secret:  
+            secretName: ssh-key # name of the Secret
+      containers:
+        - name: filemonitor-service
+          image: ""
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          volumeMounts:
+            - name: av-storage
+              mountPath: /data  
+            - name: ssh-key  
+              mountPath: /keys
+              readOnly: true
+          resources:
+            requests:
+              cpu: 20m
+              memory: 250Mi
+            limits:
+              cpu: 100m
+              memory: 500Mi
+          env:
+            # .NET Configuration
+            - name: ASPNETCORE_ENVIRONMENT
+              value: Production
+            - name: ASPNETCORE_URLS
+              value: http://+:8080
+
+            # Common Service Configuration
+            - name: Service__ApiUrl
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: API_HOST_URL
+
+            - name: Auth__Keycloak__Authority
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_AUTHORITY
+            - name: Auth__Keycloak__Audience
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_AUDIENCE
+            - name: Auth__Keycloak__Secret
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_CLIENT_SECRET
+
+            - name: Kafka__Producer__ClientId
+              valueFrom:
+                configMapKeyRef:
+                  name: filemonitor-service
+                  key: KAKFA_CLIENT_ID
+            - name: Kafka__Producer__BootstrapServers
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KAFKA_BOOTSTRAP_SERVERS
+
+            - name: Service__MaxFailLimit
+              valueFrom:
+                configMapKeyRef:
+                  name: filemonitor-service
+                  key: MAX_FAIL_LIMIT
+            - name: Service__Topics
+              valueFrom:
+                configMapKeyRef:
+                  name: filemonitor-service
+                  key: TOPICS
+            - name: Service__NotificationTopic
```

Comment:
I don't believe this service needs this.

Reply:
I removed the Service__NotificationTopic section.